src/components/Shared/CmdK.tsx

```diff
@@ -153,21 +155,21 @@ export function CmdK() {
           value={item.content}
           onMouseEnter={() => {
             eventRef.current = 'mouse'
-
             setActiveItem(index)
           }}
-          onSelect={() => {
-            if (eventRef.current === 'keyboard') {
+          onSelect={value => {
+            if (eventRef.current === 'keyboard' && activeItem !== -1) {
               return
             }
 
-            onItemSelect(item)
+            onItemSelect(activeItem === -1 ? results[index] : item)
           }}
         >
           <Button
             size="lg"
-            design="outlined"
-            className="flex flex-grow justify-between transition-colors group-data-[active=true]:bg-main group-data-[active=true]:text-on-main-variant"
+            design={activeItem === index ? 'filled' : 'outlined'}
+            intent={activeItem === index ? 'main' : 'basic'}
+            className="flex flex-grow justify-between transition-colors"
```

Comment:
I think you can remove the `group` class from the `Command.Item` component as well 😉

Reply:
solved